src/core/config.go

```diff
@@ -34,8 +34,11 @@ const LocalConfigFileName string = ".plzconfig.local"
 // for a particular machine (eg. build machine with different caching behaviour).
 const MachineConfigFileName = "/etc/plzconfig"
 
-const TestContainerDocker = "docker"
-const TestContainerNone = "none"
+const (
+	ContainerImplementationNone   = "none"
+	ContainerImplementationDocker = "docker"
+	ContainerImplementationPlz    = "plz"
+)
```

Comment:
Yeah so I'm starting to think this is structured wrong; rather than using `container` for this maybe there should be a separate property called `sandbox` or something that uses this, with `container` effectively overriding it to use a full-blown container implementation.